examples/data-objects/badge/src/helpers.ts

```diff
@@ -124,10 +124,12 @@ export const getButtonStyles = (baseColor: string): IButtonStyles => {
     };
 };
 
-export const getRelativeDate = (timestamp: Date): string => {
+export const getRelativeDate = (timestamp: string): string => {
+    const date = new Date(timestamp);
+
     // https://stackoverflow.com/questions/7641791/javascript-library-for-human-friendly-relative-date-formatting
     const delta = Math.round(
-        (new Date().getTime() - new Date(timestamp).getTime()) / 1000,
+        (new Date().getTime() - date.getTime()) / 1000,
```

Comment:
@anthony-murphy - new Date(<ISO UTC string>) parses the string, so it works until...  (see below.)